spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/controller/TaskController.java

```diff
@@ -46,18 +56,67 @@
 	@Autowired
 	private TaskDefinitionRepository repository;
 
+	@Autowired
+	@Qualifier("taskModuleDeployer")
+	private ModuleDeployer moduleDeployer;
+
+	/**
+	 * The module registry this controller will use to look up modules.
+	 */
+	private final ModuleRegistry registry;
+
+	/**
+	 * Creates a {@code TaskController} that delegates
+	 * <ul>
```

Comment:
this list order should match the ctor args (also would be more in line with the logical flow)